### ISSUE TITLE
Tip, Tip Group and Tip Manager - Provide alternative text for images - (2036570372)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5613]
### Module: 
 36 Tip, Tip Group and Tip Manager 

### Violation: 
 Provide alternative text for images 

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) 

### Instance ID: 
 2036570372 

### Severity: 
 10 

### Description: 
 [Issue]
There are images that serve a decorative purpose but are exposed to screen readers. 

[User Impact]
Screen readers may announce <img> elements with no alt attribute by their filename. This gives screen reader users irrelevant and confusing information.

[Code Reference] 
 ```
<img slot="thumbnail" src="https://placeimg.com/1000/600/nature" alt="Nature placeholder image">
 ``` 

### Note: 
 [Suggestion]
Provide alternative text for images. Non-meaningful images must be marked as decorative, which causes screen readers to ignore them. To mark an ``` <img> ``` element as decorative, set its alt attribute to "" (that is, alt="" exactly as written, with no space between the quotation marks).

[Compliant Code Example] 
 ```
<img slot="thumbnail" src="https://placeimg.com/1000/600/nature" alt="">
 ```
 

### Media Type: 
 Images 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.
